app/javascript/controllers/new_message_controller.js

```diff
@@ -47,4 +47,11 @@ export default class extends Controller {
     this.focusInput()
     event.preventDefault()
   }
+
+  unfocusKeydown(event) {
+    if (event.key === "Escape" && event.target.tagName === "TEXTAREA") {
```

Comment:
Since you're attaching this handler with `keydown.esc->new-message#unfocusKeydown` you shouldn't need to check that it's an Escape key.

Reply:
Ohhh, I didn't understand why you did that with focusKeydown so I was just blindly following the pattern, but now that I read the comment you made it clear: it's because "/" is a legit character that can be typed int he input so it needs to be checked.